AzureDevOpsTemplates/Build/StepTemplates/dfc-arm-build.yml

```diff
@@ -28,17 +29,9 @@ steps:
     scriptPath: ${{ parameters.TestArmTemplateScriptFilePath }}
     scriptArguments: -ParameterFile '${{ parameters.ArmTemplateRoot }}\test-parameters.json' -TemplateFile '${{ parameters.ArmTemplateRoot }}\template.json' -Verbose
     azurePowerShellVersion: LatestVersion
-- task: CopyFiles@2
-  displayName: 'Copy Arm Templates to Staging'
-  inputs:
-    SourceFolder: '${{ parameters.ArmTemplateRoot }}\'
-    Contents: |
-      **
-      !AzureDevOps\**
-    TargetFolder: $(build.artifactstagingdirectory)/Azure/ArmTemplates
+    
 - task: PublishPipelineArtifact@0
   displayName: Publish Pipeline Artifact
-  condition: ${{ parameters.PublishPipelineArtifact }}
   inputs:
-    targetPath: $(build.artifactstagingdirectory)/Azure/ArmTemplates
```

Comment:
With pipeline artifacts, you no longer need to copy things to the artifacts directory!